src/http/client/response.cr

```diff
@@ -75,8 +93,8 @@ class HTTP::Client::Response
     end
   end
 
-  def self.mandatory_body?(status_code) : Bool
-    !(status_code / 100 == 1 || status_code == 204 || status_code == 304)
+  def self.mandatory_body?(status : HTTP::Status) : Bool
+    !(status.informational? || status == HTTP::Status::NO_CONTENT || status == HTTP::Status::NOT_MODIFIED)
```

Comment:
```suggestion
    !(status.informational? || status.no_content? || status.not_modified?)
```

Reply:
Oh cool, didn't realise you got these methods for free. Will update.